src/lightning_trainable/trainable.py

```diff
@@ -31,6 +32,7 @@ class TrainableHParams(HParams):
     profiler: str | Profiler | None = None
     lr_scheduler: str | None = None
     num_workers: int = 4
+    name: str = "lightning_logs"
```

Comment:
`log_dir` is currently a parameter to `__init__` as it does not affect the behavior of model or training. I think we should make it consistent.

Reply:
Or maybe what did you want to achieve with `name` when you have `log_dir`?